src/app.ts

```diff
@@ -53,6 +64,10 @@ export interface AppData {
     status: AppStatus;
 }
 
+
+/**
+ * Type for AppCreateOptions
```

Comment:
please define properties of the interface. See Typedef here https://devhints.io/jsdoc

also add a description of what this is used for. e.g. "Options for creating a new app"